src/main/java/com/sourcegraph/cody/SubscriptionPanel.kt

```diff
@@ -2,10 +2,38 @@ package com.sourcegraph.cody
 
 import com.intellij.ide.BrowserUtil
 import com.intellij.ide.ui.laf.darcula.ui.DarculaButtonUI
+import com.intellij.ui.SeparatorComponent
+import com.intellij.ui.dsl.builder.BottomGap
+import com.intellij.ui.dsl.builder.TopGap
 import com.intellij.ui.dsl.builder.panel
+import com.sourcegraph.common.UpgradeToCodyProNotification
 import com.sourcegraph.config.ConfigUtil
 
 fun createSubscriptionTab(isCurrentUserPro: Boolean) = panel {
+  val chatLimitError = UpgradeToCodyProNotification.chatRateLimitError.get()
+  val autocompleteLimitError = UpgradeToCodyProNotification.autocompleteRateLimitError.get()
+  if (!isCurrentUserPro) {
```

Comment:
So the layout is the same in all 3 options and only difference is text?
If that is the case could you please refactor it so only changing element (label) is determined by ifs chain, and the layout is always the same and just uses proper label text?

Reply:
Actually I just noticed that cell separator component in one of the cases.
What is that needed for?